recipes/diffpy.labpdfproc/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{%- set name = "diffpy.labpdfproc" -%}
```

Comment:
No need for the dashes in these lines. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L8-L9

Reply:
Fixed:

```
{% set name = "diffpy.labpdfproc" %}
{% set version = "0.1.2" %}
```